recipes/inversify-cpp/all/conanfile.py

```diff
@@ -48,3 +48,8 @@ def package(self):
 
     def package_id(self):
         self.info.header_only()
+
+    def package_info(self):
+        self.cpp_info.names["cmake_find_package"] = "inversify-cpp"
+        self.cpp_info.names["cmake_find_package_multi"] = "inversify-cpp"
+        self.cpp_info.names["pkg_config"] = "inversify-cpp"
```

Comment:
I did not find these defined in your project + they are the default from Conan
We only explicit set the ones defined in the project. 

I think these can be removed